src/code-of-conduct.md

```diff
@@ -1,37 +1,5 @@
 Code of Conduct
```

Comment:
Just for future reference, I've found this, which seems to be fairly recent so could be useful :) https://docs.google.com/document/d/1Zyr50nJvuX4b9UBN4S9KXaOYLdxP4X1ir9H3YulHa7A/edit

Reply:
great, thanks! We can use it.